BAZProjectC1/Components/CarouselMovies/CarouselMoviesDelegate.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  CarouselMoviesDelegate.swift
+//  BAZProjectC1
+//
+//  Created by 1044336 on 21/09/22.
+//
+
+import Foundation
+
+internal protocol CarouselMoviesDelegate: AnyObject {
+    func movieSelected(position:Int)
+}
+internal protocol CarouselMoviesPositionDelegate: AnyObject {
+    func getCurrentItem(index:IndexPath)
+}
```

Comment:
El keyword `internal` es el default, así que no es necesario escribirlo. Considera renombrar los métodos como te mencioné anteriormente.


```suggestion
protocol CarouselMoviesDelegate: AnyObject {
    func movieSelected(position:Int)
}

protocol CarouselMoviesPositionDelegate: AnyObject {
    func getCurrentItem(index:IndexPath)
}
```